priv/stats_descriptions.cfg

```diff
@@ -51,8 +51,8 @@
     {desc, <<"number of _local document write operations">>}
 ]}.
 {[couchdb, httpd, bulk_requests], [
-    {type, counter},
-    {desc, <<"number of bulk requests">>}
+    {type, histogram},
+    {desc, <<"distribution of bulk request sizes">>}
```

Comment:
I think this metric should be renamed. Now it's not about how much bulk requests were made, but how many documents were passed in bulk. And request size here confusing: without reading the code I would say that this is amount of bytes passed with request.

Suggestion: `[couchdb, httpd, bulk_docs]` and `distribution of amount docs stored in bulk`.

Sounds good?